examples/reading_events.rs

```diff
@@ -0,0 +1,188 @@
+use eventstore::{
+    All, Client, Credentials, EventData, ExpectedVersion, Origin, ReadAllOptions, ReadResult,
+    ReadStreamOptions,
+};
+use futures::TryStreamExt;
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use uuid::Uuid;
+
+#[derive(Serialize, Deserialize, Debug)]
+struct TestEvent {
+    pub id: String,
+    pub important_data: String,
+}
+
+type Result<A> = std::result::Result<A, Box<dyn Error>>;
+
+pub async fn read_from_stream(client: &Client) -> Result<()> {
+    // region read-from-stream
+    let result = client
+        .read_stream("some-stream", &Default::default(), All)
```

Comment:
**FIX:** This sample should also show how to pass direction and stream position. (I know that they're default, but it's the first sample so it should be explicit).

Reply:
Done